cli/pkg/cmd/root.go

```diff
@@ -39,7 +40,23 @@ var versionCmd = &cobra.Command{
 	Short: "Prints version information",
 	Long:  ``,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Printf("Version: %s, GitCommit: %s\n", Version, CommitSHA1)
+		fmt.Printf("CLI version: %s, GitCommit: %s\n", Version, CommitSHA1)
+
+		wsPath, _ := cmd.Flags().GetString("workspace")
+		dir := workspace.Init(wsPath, false) // Will exit if command is invoked outside of a workspace
+		dir.LoadAiryYaml()
+		kubeCtx := kube.Load()
+		set, err := kubeCtx.GetClientSet()
+		if err != nil {
+			return
+		}
+
+		coreConfig, err := kube.GetCmData("core-config", viper.GetString("namespace"), set)
+		if err != nil {
+			fmt.Println("Warning: Unable to retrieve the version of the Airy Core instance.")
```

Comment:
Can't hurt to include the original error in this message, so that we can debug this in case a user has issues :)